src/pudl/transform/eia861.py

```diff
@@ -505,7 +639,7 @@ def backfill_ba_codes_by_ba_id(df: pd.DataFrame) -> pd.DataFrame:
         .assign(
             balancing_authority_code_eia=lambda x: x.balancing_authority_code_eia_bfilled
         )
-        .drop("balancing_authority_code_eia_bfilled", axis="columns")
+        .drop(columns="balancing_authority_code_eia_bfilled")
```

Comment:
I switched all the `df.drop()` to use `columns=` syntax since it's more readable than `axis=1`